Presentation.Web/Controllers/API/KLEController.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Web.Http;
+using Core.ApplicationServices;
+using Core.DomainModel.KLE;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models;
+
+namespace Presentation.Web.Controllers.API
+{
+    [InternalApi]
+    [RoutePrefix("api/v1/kle")]
+    public class KLEController : BaseApiController
+    {
+        private readonly IKLEApplicationService _kleApplicationService;
+
+        public KLEController(IKLEApplicationService kleApplicationService)
+        {
+            _kleApplicationService = kleApplicationService;
+        }
+
+        [HttpGet]
+        [Route("status")]
+        public HttpResponseMessage GetKLEStatus()
+        {
+            var result = _kleApplicationService.GetKLEStatus();
+
+            switch (result.Ok)
```

Comment:
@strongmindsmhs en switch med to cases... se evt. andre steder hvor result type er anvendt. Der ligger en "FromOperationFailure" du kan kalde fra BaseApiController og den kan oversætte til den rigtige fejl (i stedet for fast på forbidden)

Reply:
@mrjsawdk Done